### ISSUE TITLE
Stack with multiple connection does not write to master

### ISSUE THREAD

**Issue by sadortun** [graphaware/neo4j-php-client#143]
Hi,

When using a stack, with 
```
        $client = ClientBuilder::create()
            ->addConnection('core0', 'bolt://neo4j@localhost:7687')
            ->addConnection('core1', 'bolt://neo4j@localhost:7688')
            ->addConnection('core2', 'bolt://neo4j@localhost:7689')
            ->setMaster('core2')
            ->build();
```

The stack does not run the queries on  master, but on a randomly selected core which result in failed queries on followers.

Thanks,
Samuel

**Comment by tim-hanssen** [comment]
Yeah same issue here, we're now forcing just the writer on the transaction client create.

**Comment by ikwattro** [comment]
Fixed in 4.8.5, use 

```php
$stack->pushWrite($query, ..) 
```

This will flag the stack as write and will use the master connection.
